### ISSUE TITLE
So never pushing to origin:gh-pages

### ISSUE THREAD

**Issue by bahmutov** [Hypercubed/gh-pages-secret-poc#1]
One thing that is a little weird in this approach is the original repo having gh-pages branch that is never pushed to origin (or it would be published as pages by GitHub). So maybe rename it to something else?

**Comment by Hypercubed** [comment]
That's true.  If the original repo is also private (it doesn't need to be) then it should be safe to push a branch with a name other than `gh-pages`.  The push to private could be:

``` sh
git push private gh-pages-private:gh-pages
```

Overall I think the better setup would be to use something like [gh-pages](https://www.npmjs.com/package/gh-pages) and specify the remoteUrl and never mix the two repos.

I'm also not certain that github isn't announcing this new private repo URL somewhere...

**Comment by Hypercubed** [comment]
If someone plans to do this regularly a simple CLI wrapper for https://www.npmjs.com/package/gh-pages would be nice.

``` sh
gh-secret --init # generates a secret key, creates a .gh-secret that is `.gitignore`ed
gh-secret publish   # publishes to the repo in .gh-secret
```

**Comment by Hypercubed** [comment]
Perhaps also should create a robots.txt to avoid tracking and enable https in the GitHub settings.
